Assignment5/Assignment5.js

```diff
@@ -0,0 +1,38 @@
+const fs = require('fs');
+var read = function(readFile) {
+  return new Promise((resolve,reject) => {
+    fs.readFile(readFile, function(error, dataFromFile) {
+      if (error)
+        reject(error);
+      resolve(dataFromFile);
+      console.log("------------------------------------------------------------");
+      console.log(dataFromFile.toString());
+    });
+  });
+};
+var append = function(appendFile, dataFromFile) {
+  return new Promise((resolve,reject) => {
+    fs.appendFile(appendFile, dataFromFile.toString(), function(error) {
+      if (error)
+        reject(error);
+      resolve();
+    });
+  });
+};
+var write = function(writeFile, dataFromFile) {
+  return new Promise((resolve,reject) => {
+    fs.writeFile(writeFile, dataFromFile, function(error) {
+      if (error)
+        reject(error);
+      resolve();
+    });
+  });
+};
+
+read('file1.txt')
+ .then(result => { return append('file2.txt', result) })
```

Comment:
no need to write curly braces and explicit `return`  statement. Use single statement arrow function syntax.